modules/authentication/proc-enabling-user-authentication-with-github-as-an-auxiliary-authentication-provider.adoc

```diff
@@ -3,13 +3,15 @@
 [id="enabling-user-authentication-with-github-as-an-auxiliary-authentication-provider"]
 = Enabling user authentication with GitHub as an auxiliary authentication provider
 
-To allow users to access GitHub templates or plugins that require GitHub authentication, configure GitHub as an auxiliary authentication provider. This method relies on a primary authentication provider for user identity management, and skips resolving user identity from this provider.
+To allow users to access GitHub templates or plugins that require GitHub authentication, configure GitHub as an auxiliary authentication provider.
+This method relies on a primary authentication provider for user identity management, and skips resolving user identity from this provider.
```

Comment:
I'd reword this for added clarity. I'm not sure about using AND to describe the relationship between these two statements

if I understand it correctly, you skips resolving user identity from a provider BECAUSE you select it as a primary auth provider. is that right?

Reply:
ok, let's remove the comma before "and" (that's what initially thre me off)